gulp/fabric.js

```diff
@@ -567,6 +569,35 @@ gulp.task('build-components-page', ['clean-samples', 'build-component-data', 'bu
 // gulp.task('index-build-all', ['build-components-page']);
 
 //
+// Packaging tasks
+// ----------------------------------------------------------------------------
+gulp.task('nuget-pack', function(callback) {
+    nugetpack({
+            id: "OfficeUIFabric",
+            version: pkg.version,
+            authors: "Microsoft Corporation",
+            owners: "Microsoft Corporation",
+            description: "Fabric is a responsive, mobile-first, front-end framework, designed to make it quick and simple for you to create web experiences using the Office Design Language. It’s easy to get up and running with Fabric—whether you’re creating a new Office experience from scratch or adding new features to an existing one.",
+            summary: "The front-end framework for building experiences for Office and Office 365.",
+            language: "en-us",
+            projectUrl: "https://github.com/OfficeDev/Office-UI-Fabric",
+            licenseUrl: "https://github.com/OfficeDev/Office-UI-Fabric/blob/master/LICENSE",
+            copyright: "Copyright (c) Microsoft Corporation",
+            requireLicenseAcceptance: true,
+            tags: "Microsoft UI Fabric CSS",
+            outputDir: paths.distPackages
+        },
+
+        [
+            {src: paths.distCSS, dest: "/content/css/"},
+            {src: paths.distJS, dest: "/content/scripts/"}
```

Comment:
Is it possible to add some of the other folders from "dist" in the package? For example, we also publish demos and individual CSS files for each component, which would be useful to distribute here as well. Thanks!